### PR TITLE
client: update to Zeebe 8.1.12

### PR DESCRIPTION
**Pull request by jaeger401**
This PR:

* Imports the GRPC proto definition from Zeebe 8.1.12 (the latest 8.1.x version available).
  * Note: the README says to use the proto definition from [Zeebe's develop branch](https://raw.githubusercontent.com/zeebe-io/zeebe/develop/gateway-protocol/src/main/proto/gateway.proto), but that branch has already moved on to versions beyond 8.1. I pulled the proto definition from [their 8.1.12 tag](https://raw.githubusercontent.com/camunda/zeebe/clients/go/v8.1.12/gateway-protocol/src/main/proto/gateway.proto) instead.
* Updates the Rakefile and README to refer to Zeebe 8.1.12.
* Updates the README to refer to the new `camunda` org instead of `zeebe-io` for the Zeebe repo.
* Updates `gen-grpc.sh` to work across platforms, since not all versions of `sed` (particularly the Mac OS X version) supply a default extension for the `-i` switch.
* ❗ **Updates this gem's version to 0.19.0.** ❗
  * After merging it should be ready to release immediately, unless you want to update the date in the CHANGELOG first. I supplied tomorrow's date as a placeholder.

Reasons:

We are seeing sporadic issues with jobs not successfully starting/completing with 8.1.12, and we'd like to rule out possible version incompatibility.

### CODE REVIEW

File: lib/zeebe/client/version.rb
Comment (rusterholz) [inline lib/zeebe/client/version.rb]: @jaeger401 Is this meant to be `0.19.0` as noted in the changelog?

**Review by rusterholz — APPROVED** [review]

File: lib/zeebe/client/version.rb
Comment (jaeger401) [reply lib/zeebe/client/version.rb]: Fixed!

**Comment by cmur2** [comment]
Hey folks, thanks for bringing this up - I'll look into it and release it then :+1:

**Comment by cmur2** [comment]
I just noted that the https://github.com/CLAassistant is not happy yet according to the comment in https://github.com/zeebe-io/zeebe-client-ruby/pull/83#issuecomment-1572776638 - need to have that before merging, sorry.

**Comment by cmur2** [comment]
I realized that version 0.18.0 was not released yet so I'll use my local update script to put Zeebe 8.1.x support and release it as 0.18.0 of this gem.

**Comment by cmur2** [comment]
Version 0.18.0 of this gem supporting Zeebe 8.1.x was released. I'll also push version 0.19.0 supporting Zeebe 8.2.x.

**Comment by cmur2** [comment]
Thanks for your input again! 🙇🏼

**Comment by jaeger401** [comment]
Thanks for the assistance, @cmur2! Much appreciated. I was waiting for approval from our legal department on the CLA, but I guess I don't have to worry about that anymore!

**Pull request closed without merging**
